pkg/placement/membership.go

```diff
@@ -11,13 +11,13 @@ import (
 
 	"github.com/dapr/dapr/pkg/placement/raft"
 	v1pb "github.com/dapr/dapr/pkg/proto/placement/v1"
+	"google.golang.org/grpc/peer"
 )
 
 const (
-	commandRetryInterval = 20 * time.Millisecond
-	commandRetryMaxCount = 3
```

Comment:
Is the retry not needed anymore?

Reply:
Retry will be redundant here. 

For upsert member operation, each runtime will send heartbeat every 1 second. so if the incoming member(runtime) was not in raft state, heartbeat will retry to upsert member.

For remove member operation, faultyDetectTimer will run periodically. if remove operation is failed, the next tick of faultyDetectTimer will retry to remove the problematic member.

Please let me know if you still need retry operation here..